package.json

```diff
@@ -333,23 +334,25 @@
     "ts-loader": "6.2.2",
     "typesafe-actions": "2.2.0",
     "typescript": "4.4.4",
+    "typescript-plugin-css-modules": "^3.4.0",
     "typescript-plugin-styled-components": "2.0.0",
     "url-loader": "1.0.0",
     "webpack": "4.46.0",
     "webpack-cli": "3.3.12"
   },
   "dependencies": {
     "@brave/react-virtualized-auto-sizer": "^1.0.4",
+    "@glif/filecoin-wallet-provider": "2.0.0-alpha.14",
     "@ledgerhq/hw-app-eth": "6.24.1",
     "@ledgerhq/hw-transport-webhid": "6.24.1",
-    "@glif/filecoin-wallet-provider": "2.0.0-alpha.14",
     "@types/jszip": "^3.1.6",
     "@types/parse-torrent": "^5.8.3",
     "@types/react-router-dom": "^5.1.9",
     "@types/webtorrent": "^0.109.0",
     "array-move": "^2.2.1",
     "bignumber.js": "^9.0.2",
     "bluebird": "^3.5.1",
+    "classnames": "^2.3.1",
```

Comment:
as mentioned in sec review i'm more in favor of just adding a new utility function instead of increasing our dependencies